src/modules/meal/mealplan/mealplan.component.scss

```diff
@@ -41,3 +41,19 @@
     height: $size;
   }
 }
+
+.title-container {
+  display: flex;
+  align-items: start;
+  align-content: start;
```

Comment:
This doesn't do anything 
![image](https://github.com/jmesserli/svmeal-ui/assets/613138/eec89570-53db-4d91-a31a-10aca8160068)

Reply:
Thanks a lot for your valuable input :) I made the appropriate change.